<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iPainter</class>
 <widget class="QWidget" name="iPainter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frame
{
	
	
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>768</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1200</width>
       <height>800</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frame
{
	border:1px solid black;
	background-color: rgb(255, 255, 255);
}
QPushButton
{
	
	font: 75 14pt &quot;华康娃娃体W5(P)&quot;;
	
	color: rgb(133, 145, 255);
	background-color: rgba(145,111, 155, 53);
	border:none;
	border-radius:8px;
}
QPushButton:pressed
{
	
	background-color: rgba(145,111, 155, 83);
	border:none;
	border-radius:8px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="erase">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>91</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Eraser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="blackPen">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>152</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Pen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancle">
      <property name="geometry">
       <rect>
        <x>758</x>
        <y>51</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Cancle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteFile">
      <property name="geometry">
       <rect>
        <x>664</x>
        <y>51</y>
        <width>77</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Delet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newFile">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>52</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
     <widget class="QPushButton" name="open">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>52</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>52</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="paintfood">
      <property name="geometry">
       <rect>
        <x>484</x>
        <y>51</y>
        <width>75</width>
        <height>42</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Print-L</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>51</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>955</x>
        <y>20</y>
        <width>180</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	
	
	color: rgb(255, 255, 255);
	border-radius:12px;
	
	
	font: 75 18pt &quot;华康娃娃体W5&quot;;
	background-color: rgb(187, 223, 255);

}</string>
      </property>
      <property name="text">
       <string>Painter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>136</y>
        <width>1021</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame
{
	border:1px solid black;
	border-radius:12px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="PaintArea" name="painter_area" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
	
	
	background-color: rgba(255, 255, 255,3);
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnList">
      <property name="geometry">
       <rect>
        <x>399</x>
        <y>52</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>List</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCount">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>80</y>
        <width>180</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	
	
	color: rgb(255, 255, 255);
	border-radius:12px;
	
	
	font: 75 18pt &quot;华康娃娃体W5&quot;;
	background-color: rgb(187, 223, 255);

}</string>
      </property>
      <property name="text">
       <string>List count:0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>261</y>
        <width>101</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget
{
	border:1px solid black;
	border-radius:3px;
}
</string>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRemove">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>222</y>
        <width>94</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPrintC">
      <property name="geometry">
       <rect>
        <x>576</x>
        <y>53</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Print-C</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintArea</class>
   <extends>QWidget</extends>
   <header>paintarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
